File: mario.py
from cs50 import get_int 
import sys


while True:
    try:
        height = get_int("Height: ")
        if (height <= 8) and (height >= 0):
            break
    except:
            continue
        
    
spaces= 1
for x in range(height):
    for i in range(height - spaces):
        print(" ", end="")
    for i in range(height-(height-spaces)):
        print("#", end="")
    print(" ", end="")
    
    for i in range(height-(height-spaces)):
        print("#", end="")

    print()
    spaces += 1
       
spaces= 1

File: mario.c
#include <stdio.h>
#include <cs50.h>
int main(void)
{

int n;
do
{
n= get_int("tamanho: ");
}
while (n < 1|| n>8);

for(int i=0; i < n; i++)
{
      for(int j=0; j < n; j++)
    {
       if(i+j < n-1)
       printf(" ");
       else
       printf("#");

}


printf("\n");


}
}